### PR TITLE
Fail if --bundle is specified but no bundler

### PR DESCRIPTION
**Pull request by Mitko-Kerezov**
If `--bundle` is passed a bundling plugin needs to be present to process the option, else undefined behavior will follow, especially if one makes use of plugins like `nativescript-dev-typescript` that have logic regarding `--bundle` option.

Part of https://github.com/NativeScript/NativeScript/issues/5129

Ping @PanayotCankov @dtopuzov @MartoYankov

### CODE REVIEW

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

File: lib/commands/base-bundler.ts [hunk 1]
```diff
@@ -0,0 +1,14 @@
+export abstract class BundleBase {
+	constructor(protected $projectData: IProjectData,
+		protected $errors: IErrors,
+		protected $options: IOptions) {
+		this.$projectData.initializeProjectData();
+	}
+
+	protected validateBundling(): void {
+		const bundlerPluginName = "nativescript-dev-webpack";
```
Comment (KristianDD) [inline lib/commands/base-bundler.ts]: Can't we make this a bit more generic and use the option passed to --bundle to get the actual plugin from a collection?

**Comment by dtopuzov** [comment]
run ci

**Comment by Mitko-Kerezov** [comment]
Ping @KristianDD with a fixed comment. 
Merge after https://github.com/telerik/mobile-cli-lib/pull/1032

**Comment by Mitko-Kerezov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Comment by dtopuzov** [comment]
run ci

**Pull request merged**